frontend/src/entry-forms/Transaction.svelte

```diff
@@ -21,12 +21,18 @@
     entry.postings = entry.postings.concat(emptyPosting());
   }
 
+  $: suggestions = Object.keys($closing_dates).filter(
```

Comment:
we should filter the (ranked) array of accounts here

Reply:
You mean using `$accounts` instead of `$closing_dates`? I tried that first, but had problems getting it to work.
However, if we now want to use `$account_details`, should I still start with `$accounts`?

Reply:
Yes to preserve the ranked order of `$accounts`, we should still start with `$accounts` and filter that to all non-closed accounts (with the help of `$account_details`)